src/python/WMCore/Services/DBS/DBSReader.py

```diff
@@ -17,17 +17,13 @@ def DBSReader(endpoint, **kwargs):
     
     msg = ''
     try:
-        dbs = _getDBS3Reader(endpoint, **kwargs)
+        dbs = DBS3Reader(endpoint, **kwargs)
```

Comment:
This is the set of changes that I am wondering why you need to make it. Granted, I want to get rid of all this with what @jha2 is doing, but I don't see what this has to do with MariaDB.